openxr/src/generated.rs

```diff
@@ -3515,6 +4196,92 @@ pub(crate) mod builder {
         }
     }
     #[repr(transparent)]
+    pub struct BindingModificationBase<'a, ATY: ActionTy> {
+        _inner: sys::BindingModificationBaseHeaderKHR,
+        _marker: PhantomData<&'a ATY>,
+    }
+    #[derive(Copy, Clone)]
+    #[repr(transparent)]
+    pub struct InteractionProfileAnalogThresholdVALVE<'a, ATY: ActionTy> {
+        inner: sys::InteractionProfileAnalogThresholdVALVE,
+        _marker: PhantomData<&'a ATY>,
+    }
+    impl<'a, ATY: ActionTy> InteractionProfileAnalogThresholdVALVE<'a, ATY> {
+        #[inline]
+        pub fn new() -> Self {
+            Self {
+                inner: sys::InteractionProfileAnalogThresholdVALVE {
+                    ty: sys::StructureType::INTERACTION_PROFILE_ANALOG_THRESHOLD_VALVE,
+                    ..unsafe { mem::zeroed() }
+                },
+                _marker: PhantomData,
+            }
+        }
+        #[doc = r" Initialize with the supplied raw values"]
+        #[doc = r""]
+        #[doc = r" # Safety"]
+        #[doc = r""]
+        #[doc = r" The guarantees normally enforced by this builder (e.g. lifetimes) must be"]
+        #[doc = r" preserved."]
+        #[inline]
+        pub unsafe fn from_raw(inner: sys::InteractionProfileAnalogThresholdVALVE) -> Self {
+            Self {
+                inner,
+                _marker: PhantomData,
+            }
+        }
+        #[inline]
+        pub fn into_raw(self) -> sys::InteractionProfileAnalogThresholdVALVE {
+            self.inner
+        }
+        #[inline]
+        pub fn as_raw(&self) -> &sys::InteractionProfileAnalogThresholdVALVE {
+            &self.inner
+        }
+        #[inline]
+        pub fn action(mut self, value: &'a Action<ATY>) -> Self {
```

Comment:
Could the type variable be pushed down to this setter, rather than associated with the entire struct?